### PR TITLE
[lcn] Fix status message of RollerShutter inverting

### LINKED ISSUE #8680: [lcn] invertUpDown parameter for rollershutter - icon displayed for open/close seems not inverted
## Expected Behavior
When pressing the DOWN/UP button on an inverted rollershutter, the  icon shoud display the closed/open icon. 

## Current Behavior
Current behavior is inverted.

## Steps to Reproduce (for Bugs)
1. Configure a rollershutter writing invertUpDown
2. Press down button

## Context
This is counter-intuitive

## Your Environment
* Version used: 2.5.9
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...): Debian/raspberry pi
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): mobile apps ios/android

### PR DESCRIPTION
**Pull request by fwolter**
The status of RollerShutter was not inverted when invertUpDown=true. Fixes #8680

Signed-off-by: Fabian Wolter <github@fabian-wolter.de>

### CODE REVIEW

File: bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleHandler.java [hunk 1]
```diff
@@ -130,7 +131,8 @@ public void initialize() {
                 }
 
                 // Initialize inversion converter
-                if (invertConfig instanceof Boolean && invertConfig.equals(true)) {
+                if (invertState instanceof Boolean && invertState.equals(true)
+                        || invertUpDown instanceof Boolean && invertUpDown.equals(true)) {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleHandler.java:135]: Minor optimization.
```suggestion
                if (Boolean.TRUE.equals(invertState) || Boolean.TRUE.equals(invertUpDown)) {
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleHandler.java (hunk 1 above)
Comment (fwolter) [reply bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleHandler.java]: True! Done.

**Review by cpmeister — APPROVED** [review]

**Comment by cpmeister** [comment]
Can you create another PR for the 2.5 branch? My git repo isn't in a good state to change branches right now.

**Comment by fwolter** [comment]
I'll do the cherry-picking.

**Pull request merged**